src/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs/promises');
+const { sep, basename } = require('path');
+
+const fileParam = process.argv[2];
+const endpointParam = process.argv[3];
+
+moveFile(fileParam, endpointParam);
```

Comment:
The script should validate the input parameters and handle potential errors. Currently, there are no checks to ensure the file or endpoint parameters are provided.